pkg/cable/libreswan/libreswan.go

```diff
@@ -157,7 +157,6 @@ func (i *libreswan) GetName() string {
 func (i *libreswan) Init() error {
 	// Write the secrets file:
 	// %any %any : PSK "secret"
-	// TODO Check whether the file already exists
```

Comment:
Do we want os.Create to overwrite existing file or use existing? Rest of code implies okay to overwrite, but just want to be sure we don't lose intent of the TODO.

Reply:
I'm not sure of the original intent of the TODO but I think it could only use existing if it contained the same PSK (which is most of the time) but it seems simpler to just overwrite each time. This code and the TODO has been the same and working for 5 years so I see no compelling reason to try to change it, and thus no reason to keep the TODO (it's flagged by Sonar).

Reply:
os.Create will either create or truncate the file if it already exists. Since we always want to use the latest PSK that is configured as part of the `submariner-gateway` pod, we should be good.